<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>747</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>747</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MagicSocketDebugger</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="widget_1" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>500</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255)</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QTreeWidget" name="treeWidget">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="itemsExpandable">
          <bool>true</bool>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <property name="columnCount">
          <number>1</number>
         </property>
         <column>
          <property name="text">
           <string>Connect</string>
          </property>
         </column>
         <item>
          <property name="text">
           <string>TCP客户端</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>TCP服务端</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>WebSocket客户端</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>WebSocket服务端</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="createTcpClient"/>
   <addaction name="createTcpServer"/>
   <addaction name="createWsClient"/>
   <addaction name="createWsServer"/>
   <addaction name="deleteConnection"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>语言</string>
    </property>
    <addaction name="actionLanguageChinese"/>
    <addaction name="actionLanguageEnglish"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>其它</string>
    </property>
    <addaction name="actionFeedback"/>
    <addaction name="actionVersion"/>
   </widget>
   <addaction name="menuLanguage"/>
   <addaction name="menu"/>
  </widget>
  <action name="createTcpClient">
   <property name="text">
    <string>创建TcpConnect</string>
   </property>
  </action>
  <action name="createTcpServer">
   <property name="text">
    <string>创建Tcp服务器</string>
   </property>
  </action>
  <action name="deleteConnection">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="createWsClient">
   <property name="text">
    <string>创建WebSocketConnect</string>
   </property>
  </action>
  <action name="createWsServer">
   <property name="text">
    <string>创建WebSocket服务器</string>
   </property>
  </action>
  <action name="actionLanguageChinese">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>简体中文</string>
   </property>
  </action>
  <action name="actionLanguageEnglish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionFeedback">
   <property name="text">
    <string>问题反馈</string>
   </property>
  </action>
  <action name="actionVersion">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>V1.2.6</string>
   </property>
   <property name="toolTip">
    <string>V1.2.6</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
